recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+        self._cmake.definitions["UA_DEBUG_DUMP_PKGS"] = self.options.debug_dump_pkgs
+        self._cmake.definitions["UA_ENABLE_HARDENING"] = self.options.enable_hardening
+        self._cmake.definitions["OPEN62541_VERSION"] = self.version
+        self._cmake.definitions["UA_DEBUG" ] = self.settings.build_type == "Debug"
+        self._cmake.definitions["UA_ENABLE_WEBSOCKET_SERVER"] = self.options.with_libwebsockets
+        self._cmake.definitions["UA_ENABLE_SUBSCRIPTIONS_ALARMS_CONDITIONS"] = self.options.enable_subscriptions_alarms_conditions
+        self._cmake.definitions["UA_ENABLE_PARSING"] = self.options.enable_parsing
+        self._cmake.definitions["UA_ENABLE_DA"] = self.options.enable_da
+        self._cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP_XDP"] = self.options.enable_pubsub_eth_uadp_xdp
+        self._cmake.definitions["UA_ENABLE_PUBSUB_ETH_UADP_ETF"] = self.options.enable_pubsub_eth_uadp_etf
+        self._cmake.definitions["UA_ENABLE_PUBSUB_MQTT"] = self.options.enable_pubsub_mqtt
+        self._cmake.definitions["UA_FORCE_32BIT"] = not(self.settings.arch == "armv8" or self.settings.arch == "x86_64")
+        self._cmake.definitions["UA_FORCE_CPP"] = self.options.force_cpp
+        self._cmake.definitions["UA_ENABLE_TYPEDESCRIPTION"] = self.options.enable_typedescription
+        self._cmake.definitions["UA_NAMESPACE_ZERO"] = self.options.namespace_zero
+
+        self._cmake.definitions["UA_BUILD_EXAMPLES"] = False
+        self._cmake.definitions["UA_BUILD_UNIT_TESTS"] = False
+        self._cmake.verbose = True
+        self._cmake.configure(source_dir=self._source_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        for f in glob.glob(os.path.join(self.package_folder, "bin", "*.pdb")):
+            os.remove(f)
+        for f in glob.glob(os.path.join(self.package_folder, "lib", "*.pdb")):
+            os.remove(f)
+
+        self.copy(
+          pattern = "*",
+          src = os.path.join(self.package_folder, "share", "open62541", "tools"),
+          dst = os.path.join(self.package_folder, "bin", "tools")
+        )
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+
+    def package_info(self):
+        self.info.options.multithreaded = self._ua_multithreaded()
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
In `package`, you remove `lib/cmake` and `lib/pkgconfig`.
Please check + add the `cmake_find_package/cmake_find_package_multi/pkg_config` names.

In the future, `.pc` will files might be created only for those recipes that set `pkg_config` explicitly. So that one should be set anyhow.